### PR TITLE
add extensions for Tag object

### PR DESCRIPTION
**Pull request by kkolur**
#### References to other Issues or PRs
[Issue #2942](https://github.com/grpc-ecosystem/grpc-gateway/issues/2942)

#### Have you read the [Contributing Guidelines](https://github.com/grpc-ecosystem/grpc-gateway/blob/master/CONTRIBUTING.md)?
YES

#### Brief description of what is fixed or changed

Added support for extensions in Tag object

### CODE REVIEW

File: protoc-gen-openapiv2/options/openapiv2.proto [hunk 1]
```diff
@@ -534,6 +534,7 @@ message Tag {
   string description = 2;
   // Additional external documentation for this tag.
   ExternalDocumentation external_docs = 3;
+  map<string, google.protobuf.Value> extensions = 4;
```
Comment (johanbrandhorst) [inline protoc-gen-openapiv2/options/openapiv2.proto:537]: Please add a comment to explain how to use this (e.g., the convention of using a `x-` prefix for the keys). A link to the extensions page of the openapiv2 spec would be nice.

Comment (kkolur) [reply protoc-gen-openapiv2/options/openapiv2.proto:555]: @johanbrandhorst Done!

**Comment by kkolur** [comment]
@johanbrandhorst can we cut a new release after this PR? That would be great 🎉

File: protoc-gen-openapiv2/options/openapiv2.proto [hunk 2]
```diff
@@ -97,6 +97,9 @@ message Swagger {
   repeated Tag tags = 13;
   // Additional external documentation.
   ExternalDocumentation external_docs = 14;
+  // Custom properties that start with "x-" such as "x-foo" used to describe
+  // extra functionality that is not covered by the standard OpenAPI Specification.
+  // See: https://swagger.io/docs/specification/openapi-extensions/
```
Comment (johanbrandhorst) [inline protoc-gen-openapiv2/options/openapiv2.proto:102]: This links to the OpenAPIv3 spec page, please update the link to https://swagger.io/docs/specification/2-0/swagger-extensions/.

**Review by johanbrandhorst — CHANGES_REQUESTED** [review]
Almost there, just a small doc update!

**Pull request merged**